bin/reth/src/args/secret_key.rs

```diff
@@ -16,10 +16,8 @@ pub enum SecretKeyError {
     SecretKeyDecodeError(#[from] SecretKeyBaseError),
     #[error("Failed to create parent directory {dir:?} for secret key: {error}")]
     FailedToCreateSecretParentDir { error: io::Error, dir: PathBuf },
-    #[error("Failed to write secret key file {secret_file:?}: {error}")]
-    FailedToWriteSecretKeyFile { error: io::Error, secret_file: PathBuf },
-    #[error("Failed to read secret key file {secret_file:?}: {error}")]
-    FailedToReadSecretKeyFile { error: io::Error, secret_file: PathBuf },
+    #[error(transparent)]
+    SecretKeyFsPathError(#[from] FsPathError),
```

Comment:
I think these should provide sufficient context for debugging, so `FailedToWriteSecretKeyFile` is no longer necessary,lgtm

Reply:
@mattsse `FailedToCreateSecretParentDir` variant is removed but `FailedToAccessKeyFile` is still used in `get_secret_key`. I don't know if that's what you meant because there is no `FailedToWriteSecretKeyFile` instance :)